landing-page.html

```diff
@@ -99,41 +110,61 @@ <h1 class="text-white text-[44px] font-bold uppercase mb-6">Sidebar</h1>
+        <form class="mt-9 px-5">
           <input type="text" placeholder="Username" class="border border-[#D2D6DE] rounded-lg h-[38px] w-full text-[#999999] px-[13px] py-2 placeholder:text-[#999999] placeholder:text-[14px] text-[14px] focus:outline-none" />
-          <div class="border border-[#D2D6DE] rounded-lg h-[38px] mt-4 mb-4 flex">
-            <input type="password" placeholder="Password" class="w-full h-[20px] mt-2 ml-[13px] text-[#999999] placeholder:text-[#999999] placeholder:text-[14px] text-[14px] focus:outline-none" />
-            <img src="./assets/icon/eye-close.svg" alt="eye-close" height="14" width="14" class="mr-3"/>
+          <div class="border border-[#D2D6DE] rounded-lg h-[38px] mt-4 mb-4 flex items-center">
+            <input type="password" id="id_password" placeholder="Password" class="w-full h-[20px] ml-[13px] text-[#999999] placeholder:text-[#999999] placeholder:text-[14px] text-[14px] focus:outline-none" />
+            <i class="fa-solid fa-eye mr-3 text-[#999999] fa-sm cursor-pointer" id="togglePassword"></i>
           </div>
           <input type="text" placeholder="Captcha Kode" class="border border-[#D2D6DE] rounded-lg h-[38px] w-full text-[#999999] px-[13px] py-2 mb-4 placeholder:text-[#999999] placeholder:text-[14px] text-[14px] focus:outline-none" />
           <button class="h-[38px] bg-[#1976D2] rounded-lg w-full text-white text-[14px] font-semibold mb-5">Login</button>
+        </form>
+        <div class="px-5">
           <div class="w-full h-[15px] border-b border-gray-200 text-center mb-5">
             <span class="px-4 mb-4 bg-white text-[14px] font-medium text-gray-500">atau</span>
           </div>
-          <button class="h-[38px] border border-gray-300 rounded-lg w-full font-semibold mb-3 text-[14px]">Login dengan TRK</button>
-          <button class="h-[38px] border border-gray-300 rounded-lg w-full font-semibold mb-8 text-[14px]">Login dengan SSO</button>
-        </form>
+          <div class="h-[38px] border border-gray-300 rounded-lg w-full font-semibold mb-3 text-[14px] py-2 text-center">
+            <a href="#" class="font-semibold text-[14px]">Login dengan TRK</a>
+          </div>
+          <div class="h-[38px] border border-gray-300 rounded-lg w-full font-semibold mb-8 text-[14px] py-2 text-center">
+            <a href="#" class="font-semibold text-[14px]">Login dengan SSO</a>
+          </div>
+        </div>
       </div>
     </div>
+    <script>
+      // show & hide password
+      const togglePassword = document.querySelector('#togglePassword');
+   
+      const password = document.querySelector('#id_password');
+      togglePassword.addEventListener('click', function (e) {
+        // toggle the type attribute
+        const type = password.getAttribute('type') === 'password' ? 'text' : 'password';
+        password.setAttribute('type', type);
+        // toggle the eye slash icon
+        this.classList.toggle('fa-eye-slash');
+      });
+    </script>
```

Comment:
discussion:
in my opinion, it is better for the script to be placed at `head` or at the end of `body` so that it will be easier to find because it is grouped with another `script` tag

Reply:
Hello @agunghide .. Thank you for your advice.. I've fixed it. This is my commit : https://github.com/jabardigitalservice/sidebar-frontend-landing-page/pull/2/commits/96913eed29da8d6a5f5ba52018827714d79e3ed5